### DISCUSSION TITLE
sudo ./start.sh command not found

### DISCUSSION THREAD

**Discussion by Cinett87** [theyosh/TerrariumPI#740] [Q&A] [answered]
I am working on getting this prog working on my PI and all I can get is a bar it tells me no screen to be resumed and when I change cd TerrariumPI and try the sudo ./Start.sh it tells me the command is not found
![loading screen for pi](https://user-images.githubusercontent.com/115588020/195199791-6d4ae74c-7bdd-4115-9713-5187270e42cc.JPG)

**Comment by theyosh** [comment, marked as answer]
the ./start.sh is old TP3 command. For TP4 it has been a service. And you screenshot shows that it is running, and you command `sudo service terrariumpi start` is valid. But it should already be running when you restart your Raspberry PI.

So now you should be able to enter the web interface at http://[raspberry_pi_ip]:8090

**Reply by Cinett87** [reply]
at which website though that one takes me back to this page when I enter it into a PC browser

**Reply by Cinett87** [reply]
Never mind I just realized you mean IP address that's the part I was missing this whole time

**Reply by Cinett87** [reply]
okay one more question, whats the login

**Reply by theyosh** [reply]
https://theyosh.github.io/TerrariumPI/setup/#login
